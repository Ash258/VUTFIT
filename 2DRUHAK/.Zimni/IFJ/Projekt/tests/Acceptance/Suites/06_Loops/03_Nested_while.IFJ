' Nested while loop

scope
	dim a as integer = 25
	dim b as integer = 10
	do while a > (0 - 9)
		b = 0 - 12
		do while b <> 1
			print a; !": "; b;
			b = b + 1
		loop
		a = a - 3
	loop
end scope