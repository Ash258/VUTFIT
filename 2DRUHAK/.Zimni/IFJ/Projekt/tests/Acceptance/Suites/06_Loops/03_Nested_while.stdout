 25: -12 25: -11 25: -10 25: -9 25: -8 25: -7 25: -6 25: -5 25: -4 25: -3 25: -2 25: -1 25:  0 22: -12 22: -11 22: -10 22: -9 22: -8 22: -7 22: -6 22: -5 22: -4 22: -3 22: -2 22: -1 22:  0 19: -12 19: -11 19: -10 19: -9 19: -8 19: -7 19: -6 19: -5 19: -4 19: -3 19: -2 19: -1 19:  0 16: -12 16: -11 16: -10 16: -9 16: -8 16: -7 16: -6 16: -5 16: -4 16: -3 16: -2 16: -1 16:  0 13: -12 13: -11 13: -10 13: -9 13: -8 13: -7 13: -6 13: -5 13: -4 13: -3 13: -2 13: -1 13:  0 10: -12 10: -11 10: -10 10: -9 10: -8 10: -7 10: -6 10: -5 10: -4 10: -3 10: -2 10: -1 10:  0 7: -12 7: -11 7: -10 7: -9 7: -8 7: -7 7: -6 7: -5 7: -4 7: -3 7: -2 7: -1 7:  0 4: -12 4: -11 4: -10 4: -9 4: -8 4: -7 4: -6 4: -5 4: -4 4: -3 4: -2 4: -1 4:  0 1: -12 1: -11 1: -10 1: -9 1: -8 1: -7 1: -6 1: -5 1: -4 1: -3 1: -2 1: -1 1:  0-2: -12-2: -11-2: -10-2: -9-2: -8-2: -7-2: -6-2: -5-2: -4-2: -3-2: -2-2: -1-2:  0-5: -12-5: -11-5: -10-5: -9-5: -8-5: -7-5: -6-5: -5-5: -4-5: -3-5: -2-5: -1-5:  0-8: -12-8: -11-8: -10-8: -9-8: -8-8: -7-8: -6-8: -5-8: -4-8: -3-8: -2-8: -1-8:  0